faithcomesbyhearing/dbp
# FCBH-1829 Verify cache is working

# Description
- Added cache helper functions
- Added `/status/cache` endpoint to verify cache functionality
- Implemented new cache helper functions
- Verified the functionality where the cache is used

## Issue Link
Story: [FCBH-1829](https://fullstacklabs.atlassian.net/browse/FCBH-1829) 

## How Do I QA This
- Run `http://dbp.test/status/cache` and verify your local cache is working

   _On this example the minute cache value is new_ 
![image](https://user-images.githubusercontent.com/41348080/81261825-5f371080-9002-11ea-995d-1f8f6125cda7.png)

- Test the endpoints and verify that the results are being cached

**Note**: If the cache is not working on your system verify the .env `CACHE_DRIVER` value

## Checklist
